packages/pros/src/devices/controller.rs

```diff
@@ -13,17 +13,30 @@ use crate::error::{bail_on, map_errno};
 /// Holds whether or not the buttons on the controller are pressed or not
 #[derive(Default, Debug, Clone, Copy, Eq, PartialEq)]
 pub struct Buttons {
+    /// A button
     pub a: bool,
+    /// B button
     pub b: bool,
+    /// X button
     pub x: bool,
+    /// Y button
     pub y: bool,
+
+    /// Up button
     pub up: bool,
+    /// Down button
     pub down: bool,
+    /// Left button
     pub left: bool,
+    /// Right button
     pub right: bool,
+    /// Front left trigger
     pub left_trigger_1: bool,
+    /// Back left trigger
     pub left_trigger_2: bool,
+    /// Front right trigger
     pub right_trigger_1: bool,
+    /// Back right trigger
     pub right_trigger_2: bool,
```

Comment:
We should verify that these are actually the right front/back order. I can't remember off the top of my head.